app/models/task.py

```diff
@@ -1,5 +1,25 @@
 from app import db
 
-
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True, default = None)
+    goal_id = db.Column(db.Integer, db.ForeignKey("goal.goal_id"), nullable=True)
+    def to_dict(self):
+        if Task.completed_at is True:
+            task_as_dict["is_completed"] == True
```

Comment:
This line of code is a little weird to me...

If you were assigning the value to something, I would expect the assignment operator, which is `=`:

```python
task_as_dict["is_completed"] = True
```

meaning that the value of the key `"is_completed"` in the dictionary `task_as_dict` has the value `True`.

However, the expression with the equality comparison operator `==` is testing if both sides of the operator are equal to each other: "Is the value found at `task_as_dict["is_completed"]` equal to `True`?"

Let me know what questions you have on this